app/src/main/res/layout/fragment_potential_risk.xml

```diff
@@ -0,0 +1,252 @@
+                android:layout_width="40dp"
+                android:layout_height="40dp"
+                android:background="?selectableItemBackgroundBorderless"
+                app:srcCompat="@drawable/ic_close"
+                android:contentDescription="@string/close_button_content_descriptor" />
+        </LinearLayout>
+
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="20dp"
+            android:background="@drawable/bg_test_confirmation"
+            android:orientation="vertical"
+            android:padding="16dp">
+
+            <TextView
+                android:id="@+id/covid_title"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:gravity="center_horizontal|center_vertical"
+                android:text="@string/covid_title"
+                android:textSize="18sp" />
+
+            <TextView
+                android:id="@+id/risk_subtitle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+
+                android:gravity="center_horizontal|center_vertical"
+                android:text="@string/risk_subtitle"
+                android:textAppearance="@style/TextAppearance.AppCompat.Display1"
+                android:textColor="@android:color/holo_red_light" />
+
+            <TextView
+                android:id="@+id/risk_description"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:gravity="center_horizontal|center_vertical"
+                android:text="@string/risk_description"
+                android:textSize="18sp" />
+        </LinearLayout>
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="24dp"
+                android:layout_marginTop="32dp"
+                android:layout_marginBottom="16dp"
+                android:layout_weight="1"
+                android:orientation="horizontal"
+                android:layout_marginHorizontal="32dp">
+
+                <ImageView
+                    android:layout_width="72dp"
+                    android:layout_height="24dp"
+                    android:contentDescription="@string/checkmark_image_content_descriptor"
+                    android:scaleType="fitCenter"
+                    app:srcCompat="@drawable/check" />
+
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="@string/monitor_symptoms"
+                    android:textSize="18sp" />
+
+            </LinearLayout>
```

Comment:
You can use `TextView` with `drawableStart` instead of wrapping it in LinearLayout